.github/workflows/document.yml

```diff
@@ -1,4 +1,4 @@
-name: Python package
+name: Documentation
```

Comment:
I think we need to build the doc only when we push on master

Reply:
We don't want to build the doc any time a new pull request is made to master but we want to re-build the doc when the code on master changes. We should change `pull_request` to `push`

Reply:
changed